composer.json

```diff
@@ -25,7 +25,8 @@
         "doctrine/instantiator": "~1.1",
         "ocramius/package-versions": "^1.1.2",
         "ocramius/proxy-manager": "^2.1.1",
-        "symfony/console": "~3.2|~4.0"
+        "symfony/console": "~3.2|~4.0",
+        "symfony/polyfill-ctype": "~1.8"
```

Comment:
Probably better to require `"ext-ctype"` and have consumers include `"symfony/polyfill-ctype"` or any other polyfill that implements the `"provides"` clause in their `composer.json`